libs/ui/src/lib/modal/modal.component.stories.ts

```diff
@@ -15,7 +15,7 @@ export const Default = {
         <h1>{{ testModalTitle }}</h1>
         <p dataAttribute="test-content">Modal content</p>
       </ng-template>
-      <cvi-ng-button [cviNgModal]="modal" modalTitle="Some title" dataAttribute="test-button">Open modal</cvi-ng-button>
+      <cvi-ng-button [cviNgModal]="modal" modalTitle="Some title" closeable="true" dataAttribute="test-button">Open modal</cvi-ng-button>
```

Comment:
Same as in `confirmation-modal.component.stories.ts`, this attribute probably should be turned into a live Storybook control using a `closeable` arg: 
```suggestion
      <cvi-ng-button [cviNgModal]="modal" modalTitle="Some title" [closeable]="closeable" dataAttribute="test-button">Open modal</cvi-ng-button>
```